library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Test with the encounter code when you do this and see if the function of moving from one encounter to another still works, this seams to be a place where things break so it is a good place to test. I think loading bootstrap was causing some issues , not sure though. What ever the means we just need a good simple way of doing this and loading only those things that are needed.